src/main/java/ua/danit/final_project/services/storage/StorageService.java

```diff
@@ -1,6 +1,9 @@
 package ua.danit.final_project.services.storage;
 
 import org.springframework.web.multipart.MultipartFile;
+import ua.danit.final_project.entities.AbstractImage;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.AbstractImage'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=163&issues=AWbpLQnWW3Y69bz5KE67&open=AWbpLQnWW3Y69bz5KE67)

[ISSUE_KEY]: AWbpLQnWW3Y69bz5KE67